scripts/jenkins/ardana/ansible/bootstrap-vcloud-nodes.yml

```diff
@@ -17,7 +17,7 @@
 
 - name: Bootstrap virtual deployer for ardana
   hosts: "{{ ardana_env }}"
-  remote_user: ardana
+  remote_user: root
```

Comment:
this file is also used for Ardana deployments. Won't changing the user affect Ardana?

Reply:
Yes, that's true @AbelNavarro . I need to test that this doesn't break the ardana jobs. And I've also received some feedback from @flaviodsr that this is a terrible way to do it and I should create separate tasks for crowbar and ardana. Might do both, if I have time

Reply:
done